spec/controllers/subject_rule_effects_controller_spec.rb

```diff
@@ -3,27 +3,125 @@
 RSpec.describe SubjectRuleEffectsController, type: :controller do
   let(:workflow) { create :workflow }
   let(:rule) { create :subject_rule, workflow: workflow }
+  let(:credentials) do
+    fake_session admin: false, project_ids: [workflow.project_id], logged_in: true
+  end
 
   context 'as a permissioned user' do
-    before{ fake_session admin: false, project_ids: [workflow.project_id], logged_in: true }
+    before { credentials }
+
+    describe '#create' do
+      let(:create_params) do
+        {
+          subject_rule_effect: { action: 'retire_subject', config: {} },
+          workflow_id: workflow.id,
+          subject_rule_id: rule.id
+        }
+      end
+
+      it 'does not create a new effect' do
+        post :create, params: create_params, format: :json
+        expect(response.status).to eq(401)
+      end
+
+      it 'redirects to the new path in html mode' do
+        post :create, params: create_params, format: :html
+        expect(response).to redirect_to(
+          new_workflow_subject_rule_subject_rule_effect_path
+        )
+      end
+
+      it 'flashes an error message' do
+        post :create, params: create_params, format: :html
+        expect(flash[:alert]).to eq('Error creating a subject effect rule')
+      end
+
+      xit 'makes a new effect' do
```

Comment:
so long as we're adding `TODO:`s, it might be worth one here so we don't forget to come back to this one

Reply:
```suggestion
      # TODO: fix these pending specs when revisiting admin lockdown
      xit 'makes a new effect' do
```